### DISCUSSION TITLE
Control+shift+click a link to close all other tiddler except the current tiddler and the linked tiddler

### DISCUSSION THREAD

**Discussion by arunnbabu81** [crazko/krystal#42] [Ideas]
It would be nice to have the option to Control+shift+click a link to close all other tiddler except the current tiddler and the linked tiddler. This helps in compare the two related tiddlers.

there is a similar functioning view toolbar button to close other tiddlers

**Comment by arunnbabu81** [comment]
[This ](https://talk.tiddlywiki.org/t/variant-of-close-other-view-toolbar-button/2520/14?u=arunnbabu81)  is a similar implementation without using the modifier keys, but doesn’t work well with the inbuilt behaviour of krystal

**Comment by crazko** [comment]
Thanks for the suggestion. Just want to let you know that it will be low in my priorities (together with #41). I need to check how different keystrokes can affect browsers' built-in functionalities.

**Reply by arunnbabu81** [reply]
Modifier keys for ` click ` on  `wikilinks` (not external links) I suggested are just examples.

Suppose this option is `on`
![image](https://user-images.githubusercontent.com/67494083/158071996-66b49ed8-798a-4d24-89b6-4e578f330276.png)
 I need these behaviors on clicking wikilinks
1. Clicking on wikilink closes the tiddler to the right which is replaced by the linked tiddler (which is the default behavior)
2. Clicking on wikilink opens the linked tiddler in the story river without closing the tiddler to the right.
3. Clicking on wikilink opens  the linked tiddler in the maximized state.
4. Clicking on wikilink closes all other tiddlers in the story river except the linked tiddler and the tiddler with the link.

Different combination of modifier keys can be used and if possible may be configurable.
